Flick/Assets.xcassets/deselect.imageset/Contents.json

```diff
@@ -0,0 +1,23 @@
+{
+  "images" : [
+    {
+      "filename" : "deselectActive1x.png",
+      "idiom" : "universal",
+      "scale" : "1x"
+    },
+    {
+      "filename" : "deselectActive2x.png",
+      "idiom" : "universal",
+      "scale" : "2x"
+    },
+    {
+      "filename" : "deselectActive3x.png",
+      "idiom" : "universal",
+      "scale" : "3x"
```

Comment:
i started putting 1x, 2x, 3x images cuz i think it's just better practice and most phones now use 3x so it's slightly costly to always upscale from 1x to 3x

Reply:
lol ya, I think this is good. youre required to do it anyways when you launch the app. I was just gonna do it all at once at the end lmao

Reply:
we also have to make sure all of our images are kind of consistent in sizing. I think they're all kinda all over the place right now.

Reply:
maybe ask Cindy to compile all of the icons with sizes somewhere for us.